### PR TITLE
Add mailing list and group config for wg-lts

### PR DESCRIPTION
**Pull request by jeremyrickard**
Following up from the charter merge for [wg-lts](https://github.com/kubernetes/community/tree/master/wg-lts), this PR adds the wg-lts directory under `groups` and adds mailing lists, group config, and OWNER_ALIASES updates.

ref: https://github.com/kubernetes/community/issues/7259

/hold for review from @liggitt and @micahhausler as well

/area groups
/area access

### CODE REVIEW

File: groups/wg-lts/groups.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      WhoCanJoin: "ANYONE_CAN_JOIN"
+      WhoCanViewGroup: "ANYONE_CAN_VIEW"
+      WhoCanDiscoverGroup: "ANYONE_CAN_DISCOVER"
+      WhoCanPostMessage: "ANYONE_CAN_POST"
+      MessageModerationLevel: "MODERATE_NON_MEMBERS"
+      ReconcileMembers: "false"
+    owners:
+     - jeremy.r.rickard@gmail.com
+      - mhausler@amazon.com
+      - liggitt@google.com
```
Comment (ameukam) [inline groups/wg-lts/groups.yaml:34]:
```suggestion
    owners:
      - jeremy.r.rickard@gmail.com
      - mhausler@amazon.com
      - liggitt@google.com
```

Comment (MadhavJivrajani) [reply groups/wg-lts/groups.yaml:34]: also nit: would prefer alpha sorting.

File: groups/wg-lts/groups.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    members:
+      - wg-lts-leads@kubernetes.io
```
Comment (MadhavJivrajani) [inline groups/wg-lts/groups.yaml:36]: I remember a discussion where for owners, another group could not be specified, and folks would need to be listed individually. @ameukam do you know if that is true for `members` as well?

File: groups/wg-lts/groups.yaml (hunk 1 above)
Comment (jeremyrickard) [reply groups/wg-lts/groups.yaml]: I pushed a commit that applies the suggestion from @ameukam  and applied alpha sorting @MadhavJivrajani, thank you for the suggestion.

**Pull request merged**